db/migrate/20150813202625_set_missing_password_digest_in_authentications.rb

```diff
@@ -0,0 +1,15 @@
+require Rails.root.join('gems/pending/util/miq-password')
```

Comment:
This should just be `require 'miq-password'`.  No need to fully qualify because both gems/pending and gems/pending/util are on the $LOAD_PATH.